### PR TITLE
EL-4074 - Angular tree package deprecated

### PR DESCRIPTION
**Pull request by cmallon367**
#### Checklist

* [x] The contents of this PR are mine to submit. No third party code or other material is being committed.
* [x] New third party dependencies (if any) are linked via `package.json`. Third party dependencies are licensed under one of the following: Apache, MIT, BSD, Mozilla Public License, Eclipse Public License, or Oracle Binary Code License.
* [x] The code in this PR does not contain export-restricted encryption algorithms.

#### Ticket / Issue

https://portal.digitalsafe.net/browse/EL-4074

#### Description of Proposed Changes

Updated to the new package @circlon/angular-tree-component 
Removed the inappropriate importing of the css being used by the tree component
#### Documentation CI URL

https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_EL-4074-Angular-tree-package-deprecated

#### Downstream Build(s)

* [caf/ux-aspects-micro-focus](https://jenkins.swinfra.net/job/SEPG/job/caf/job/caf~ux-aspects-micro-focus~EL-4074-Angular-tree-package-deprecated~CI/)

### CODE REVIEW

**Comment by cmallon367** [comment]
Just looking for feedback to make sure I'm going about this the correct way

**Review by donna-marijne — CHANGES_REQUESTED** [review]
Also need to update the library peer dependencies, in src/package.json.

**Review by DeanHaire — APPROVED** [review]

**Review by donna-marijne — CHANGES_REQUESTED** [review]

File: BREAKING_CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # 2.1.x to 3.0
 * Angular 8 is no longer supported.
+* Angular tree component is now deprecated.
```
Comment (donna-marijne) [inline BREAKING_CHANGES.md:3]:
```suggestion
* The package for `angular-tree-component` has been renamed.
```

**Review by donna-marijne — CHANGES_REQUESTED** [review]

**Review by donna-marijne — APPROVED** [review]
Looks good, thank you.

**Review by DeanHaire — APPROVED** [review]

**Pull request merged**